### ISSUE TITLE
Change default branch to 7.x?

### ISSUE THREAD

**Issue by mjordan** [MarcusBarnes/islandora_compound_batch#9]
By convention, Islandora modules use 7.x as their default branch. Any reason Islandora Compound Batch should use master?

**Comment by MarcusBarnes** [comment]
No.  We should use the same pattern as core Islandora modules.  Thank you for pointing this out.
